test/analysis/native_deps_test.bzl

```diff
@@ -0,0 +1,42 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
+load("//rust:rust.bzl", "rust_binary")
+
+def _linkopts_test_impl(ctx):
+    env = analysistest.begin(ctx)
+
+    subject = analysistest.target_under_test(env)
+    actions = analysistest.target_actions(env)
+
+    asserts.true(env, "-lsystem_lib" in actions[0].argv)
+
+    return analysistest.end(env)
+
+linkopts_test = analysistest.make(_linkopts_test_impl)
+
+def _linkopts_test():
+    cc_library(
+        name = "native",
+        linkopts = ["-lsystem_lib"],
+        tags = ["manual"],
```

Comment:
Out of curiosity, why does this test env need these manual tags?

Reply:
Just so it's not built when building //...